src/Riok.Mapperly.Abstractions/PropertyNameMappingStrategy.cs

```diff
@@ -0,0 +1,17 @@
+namespace Riok.Mapperly.Abstractions;
+
+/// <summary>
+/// Defines the strategy to use when mapping an property to another property.
+/// </summary>
+public enum PropertyNameMappingStrategy
+{
+    /// <summary>
+    /// Matches property by its name in case sensitive manner.
```

Comment:
Would add an article: `Matches a property ...` (same for CaseInsensitive).

Reply:
will do!